tests/fltest_screen.fish

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env fish
+# -*-  mode:fish; tab-width:4  -*-
+
+function fltest_screen
+
+    echo "Context: STY var is set to 'fishline_test'"
```

Comment:
Exactly, I need to update the documentation badly about all the internals and way to you can create and extend fishline.

A lot of the work done for Fishline 3.0 missed documentation and I wanted to wait before releasing it, but in the end I preferred to have it with out-of-date documentation as it was my vision of what fishline should have been from the get go and a lot of the people that are using fishline needed some of the change of v3.0.

I am taking note about that though, to add some note about the test runner.